src/tests/MasterTests.cpp

```diff
@@ -7,40 +7,30 @@
 //
 // Screaming Udder!                              https://esss.se
 
+#include <gtest/gtest.h>
+#include <memory>
+
 #include "CommandListener.h"
 #include "JobCreator.h"
 #include "Master.h"
 #include "Msg.h"
 #include "Status/StatusReporter.h"
 #include "helpers/FakeStreamMaster.h"
 #include "helpers/KafkaWMocks.h"
-#include <gtest/gtest.h>
-#include <memory>
+#include "helpers/RunStartStopHelpers.h"
 
 using namespace FileWriter;
+using namespace RunStartStopHelpers;
 
-std::string StartCommand{R"""({
-    "cmd": "filewriter_new",
-    "broker": "localhost:9092",
-    "job_id": "1234",
-    "file_attributes": {"file_name": "output_file1.nxs"},
-    "nexus_structure": { }
-  })"""};
-
-std::string StopCommand{R"""({
-    "cmd": "filewriter_stop",
-    "job_id": "1234"
-  })"""};
-
-TEST(ParseCommandTests, IfCommandStringIsParseableThenDoesNotThrow) {
```

Comment:
Ditto about replacement of unit tests.

Reply:
The nearest equivalent now would be passing verification, which we can't do for reasons explained here https://github.com/ess-dmsc/kafka-to-nexus/blob/ef529d1bbe46c8f6aa3d216e562d7d62f827c114/src/CommandParser.cpp#L102-L104. So this test was removed.

Reply:
The two tests you have commented on, plus the "successful parsing" version of this test are the only ones removed. All the others were just updated to take flatbuffer input rather than a JSON string.

Reply:
Thanks.